site/_includes/macros/cards/frontmatter-card.njk

```diff
@@ -0,0 +1,57 @@
+{% from 'macros/card-authors.njk' import cardAuthors with context %}
+
+{% macro frontMatterCard(fm) %}
+<div class="blog-card rounded-lg pad-300 md:pad-400 width-full">
+  {% if fm.hero %}
+    {% set alt = fm.title or 'Unlabeled image' %}
```

Comment:
How often do posts not have titles?

Reply:
Unlikely often, but there are markdown files without a title. I don't filter them out, though I probably should...